app/src/main/java/org/gdg/frisbee/android/gde/GdeAdapter.java

```diff
@@ -129,21 +113,10 @@ public Person doInBackground(ViewHolder... params) {
                         Person gde = null;
 
                         if (matcher.matches()) {
-                            String plusId = matcher.group(1);
-                            gde = (Person) App.getInstance().getModelCache().get(Const.CACHE_KEY_GDE + plusId, !Utils.isOnline(mContext));
```

Comment:
This part is also leaking memory. 

When the device is only, GDE profiles are being requested live from G+ api while user scrolls. And these actually keep the ListFragment reference. When the user scrolls through tabs quickly, it doesn't let the Fragment instances to be recycled. 

getPersonSync implementation is always using the cache when available. So it will be a little bit better.